Author: Alastair McKinstry <mckinstry@debian.org>
Description: Fixes needed for autoreconf to work on Debian
 When autoreconf is run, VL_LIB_READLINE is lost from aclocal, so included in config/
 Also include AX_CHECK_COMPLER_FLAG() which is not in autoconf (just autoconf-archive)
 AX_CHECK_COMPILER_FLAGS() is obsolete
Last-Updated: 2021-01-14
Forwarded: no
Link: https://sources.debian.org/patches/silo-llnl/4.11-2/autoreconf.patch/

--- a/configure.ac
+++ b/configure.ac
@@ -195,6 +195,8 @@
 dnl This allows multiple src-dir builds within one host.
 AC_PREFIX_DEFAULT("SILO_TOP_SRC_DIR")
 
+AC_PROG_F77
+
 dnl
 dnl Handle the python module right away to determine if we need shared libs.
 dnl Ordinarily, we default to static libs
@@ -618,10 +620,10 @@
 # it is an argument to the -D argument. So, I think this is
 # just totally bogus!
 # Default to large file support
-AX_CHECK_COMPILER_FLAGS("-D_LARGEFILE_SOURCE",CFLAGS="$CFLAGS -D_LARGEFILE_SOURCE";)
-AX_CHECK_COMPILER_FLAGS("-D_LARGEFILE64_SOURCE",CFLAGS="$CFLAGS -D_LARGEFILE64_SOURCE";)
-AX_CHECK_COMPILER_FLAGS("-D_FILE_OFFSET_BITS=64",CFLAGS="$CFLAGS -D_FILE_OFFSET_BITS=64";)
-AX_CHECK_COMPILER_FLAGS("-Wdeclaration-after-statement",CFLAGS="$CFLAGS -Wdeclaration-after-statement";)
+AX_CHECK_COMPILE_FLAG("-D_LARGEFILE_SOURCE",CFLAGS="$CFLAGS -D_LARGEFILE_SOURCE";)
+AX_CHECK_COMPILE_FLAG("-D_LARGEFILE64_SOURCE",CFLAGS="$CFLAGS -D_LARGEFILE64_SOURCE";)
+AX_CHECK_COMPILE_FLAG("-D_FILE_OFFSET_BITS=64",CFLAGS="$CFLAGS -D_FILE_OFFSET_BITS=64";)
+AX_CHECK_COMPILE_FLAG("-Wdeclaration-after-statement",CFLAGS="$CFLAGS -Wdeclaration-after-statement";)
 
 #
 # Note: regardless of what the stuff above regarding large file support
